### DISCUSSION TITLE
Compartment Length/Width of A Complex Enclosure

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/fds#13007] [General]
Hi,

I'm not sure if this is the right chat to be asking this question, but I was just wondering if there is a right way of measuring the compartment length or compartment width of an enclosure with a complex geometry.

For instance, if you have a multi-room enclosure/compartment, how do you you measure the compartment width or compartment length? Are these measured as 2.4m width and 16.8m (12m + 4.8m) length (please see attached)? Or is 4.8m width and 12m length (please see attached)?

![Screenshot (12812)](https://github.com/firemodels/fds/assets/161180346/982faa51-286e-4cfb-af77-1def47aa19e4)

**Comment by drjfloyd** [comment]
If you are trying to define the overall extent of the domain, then 12 x 4.8. If you are trying to describe each compartment, then use the dimensions for each compartment.

**Reply by FireStudent321** [reply]
Thanks for the reply. The reason why I'm asking is because I was just wondering if the multi-room enclosure meet the range of validity of the "compartment fire framework". This range state that when the enclosure aspect ratio (compartment length divided by compartment width) is above 5, caution should be taken.

**Reply by TristanHehnen** [reply]
From looking at your image, it looks to me that there are three compartments: a corridor and two attached rooms. 
The ration may be intended to be determined for each room individually. For very long compartments, like the corridor, the assumption of a homogeneous smoke layer with an even thickness may not hold. It may be fine for the adjacent rooms.

**Reply by drjfloyd** [reply]
The image is from the Validaton Guide. See the first row. You are using a CFD model, not a zone model; FDS is going to predict the 3D shape and temperature profile of the layer of time unlike a zone model which assumes a perfectly stratified compartment.

<img width="865" alt="image" src="https://github.com/firemodels/fds/assets/12799217/bb64845a-b938-437a-a9e9-0f4b07d781de">

**Reply by TristanHehnen** [reply]
Yes, I had zone models in mind. Thank you for the clarification @drjfloyd !

**Reply by FireStudent321** [reply]
Thank you, @drjfloyd and @TristanHehnen.
